live-examples/html-examples/inline-text-semantics/css/kbd.css

```diff
@@ -0,0 +1,6 @@
+kbd {
+  background: #f5f5f5;
```

Comment:
Nit: Code style: indentation of 4 spaces. Also, as this specifies only a colour, use `background-color` instead of the shorthand.